ReLERNN/helpers.py

```diff
@@ -7,6 +7,114 @@
 from ReLERNN.simulator import *
 from ReLERNN.sequenceBatchGenerator import *
 
+
+def get_corrected_index(L,N):
+    idx,outN="",""
+    dist=float("inf")
+    for i in range(len(L)):
+        D=abs(N-L[i])
+        if D < dist:
+            idx=i
+            outN=L[i]
+            dist=D
+    return [idx,outN]
+
+#-------------------------------------------------------------------------------------------
+
+def get_corrected(rate,bs):
+    idx=get_corrected_index(bs["Q2"],rate)
+    CI95LO=bs["CI95LO"][idx[0]]
+    CI95HI=bs["CI95HI"][idx[0]]
+    cRATE=relu(rate+(bs["rho"][idx[0]]-idx[1]))
+    ciHI=relu(cRATE+(CI95HI-idx[1]))
+    ciLO=relu(cRATE+(CI95LO-idx[1]))
+    return [cRATE,ciLO,ciHI]
+
+#-------------------------------------------------------------------------------------------
+
+def get_index(pos, winSize):
+    y=snps_per_win(pos,winSize)
+    st=0
+    indices=[]
+    for i in range(len(y)):
+        indices.append([st,st+y[i]])
+        st+=y[i]
+    return indices
+
+#-------------------------------------------------------------------------------------------
+
+def snps_per_win(pos, window_size):
```

Comment:
Why is this (as well as a few others) a helper function as well as a method of Manager?